Index: src/configure.in
===================================================================
--- src.orig/configure.in
+++ src/configure.in
@@ -2507,12 +2507,12 @@ else
     TCLSH=tclsh
 fi
 
-AC_MSG_CHECKING([for BWidget using $TCLSH])
-if (unset DISPLAY; echo ["catch { package require bwidget }; exit [expr [lsearch [package names] BWidget] == -1]"] | $TCLSH); then
-    AC_MSG_RESULT([found])
-else
-    AC_MSG_ERROR([not found])
-fi
+#AC_MSG_CHECKING([for BWidget using $TCLSH])
+#if (unset DISPLAY; echo ["catch { package require bwidget }; exit [expr [lsearch [package names] BWidget] == -1]"] | $TCLSH); then
+#    AC_MSG_RESULT([found])
+#else
+#    AC_MSG_ERROR([not found])
+#fi
 
 HAVE_WORKING_BLT=no
 AC_MSG_CHECKING([for BLT using tclsh8.5])
