Index: src/configure.in
===================================================================
--- src.orig/configure.in
+++ src/configure.in
@@ -2829,19 +2829,19 @@ Install this version and specify --with-
     AC_CHECK_HEADERS(X11/Xmu/Xmu.h,[],[AC_MSG_ERROR([Required Xmu header missing.  Install it, or specify --disable-python to skip the parts of LinuxCNC that depend on Python])])
 fi
 
-AC_MSG_CHECKING(libgl1-mesa-dri workaround)
-
-PRELOAD_WORKAROUND=`../scripts/test-libgl-bug.sh 2>/dev/null`
-if test  $? -ne 0 ; then
+#AC_MSG_CHECKING(libgl1-mesa-dri workaround)
+#
+#PRELOAD_WORKAROUND=`../scripts/test-libgl-bug.sh 2>/dev/null`
+#if test  $? -ne 0 ; then
    # real bad, the test failed - let me know
-   AC_MSG_RESULT([test for libgl1-mesa-dri workaround failed, please file a bug])
-   exit 1
-fi
-if test "x$PRELOAD_WORKAROUND" != "x" ; then
-   AC_MSG_RESULT(required - need to preload $PRELOAD_WORKAROUND)
-else
-   AC_MSG_RESULT(not required)
-fi
+#   AC_MSG_RESULT([test for libgl1-mesa-dri workaround failed, please file a bug])
+#   exit 1
+#fi
+#if test "x$PRELOAD_WORKAROUND" != "x" ; then
+#   AC_MSG_RESULT(required - need to preload $PRELOAD_WORKAROUND)
+#else
+#   AC_MSG_RESULT(not required)
+#fi
 
 AC_SUBST([BUILD_PYTHON])
 AC_SUBST([LIBPYTHON])
